example/package.json

```diff
@@ -11,11 +11,6 @@
     "graphql": "^14.5.8",
     "graphql-tag": "^2.10.1",
     "isomorphic-unfetch": "^3.0.0",
-    "next": "9.1.1",
-    "next-urql": "file:../next-urql.tgz",
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0",
-    "react-is": "^16.11.0",
-    "urql": "^1.6.1"
```

Comment:
Removed all of these so we can guarantee resolution to a single copy of `react` in root `node_modules`.